.gitlab/newell.gitlab-ci.yml

```diff
@@ -0,0 +1,24 @@
+
+cleanup:
+ stage: .pre
+ extends:
+  - .SVC-Account-Cleanup
+
+hykkt-test-newell:
+  variables:
+    SLURM_Q: "newell_shared"
+    MY_CLUSTER: "newell"
+  extends:
+    - .cluster_test
+
+success:
+  extends:
+    - .report-status
+  stage: .post
+
+failure:
```

Comment:
I think Newell can still have a pending status posted. Maybe Deception can as well, but that would require posting the job status from within the HPC runner, which we decided wasn't ideal.

Reply:
I think would probably require a separate pending-reoport-job. Unless there is some way to just have the status be pending automatically until otherwise overridden.

Reply:
Ok - lets just have fail/success without pending for all platforms then

Reply:
I have pending included now. So all good. It's a little hacky with the pending but it works. Plus I used description to allow deception and newel be the name of the pipeline.